bundles/org.openhab.core.model.core/src/main/java/org/openhab/core/model/core/internal/folder/FolderObserver.java

```diff
@@ -60,6 +62,12 @@
 
     /* the model repository is provided as a service */
     private final ModelRepository modelRepository;
+    private static final String READYMARKER_TYPE = "dsl";
+
+    @Reference
+    /* default */ ReadyService readyService;
```

Comment:
The `FolderObserver` already uses constructor injection? Can we use it for the `ReadyService` too?